### DISCUSSION TITLE
Fossify Collective is now up!

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#158] [Announcements]
Here's the link: https://opencollective.com/fossify

This also makes credit/debit card donations possible, thank you!

**Comment by Jofi47** [comment]
Hello, I think it's great that you want to continue the project. It would be nice if you could set up another payment option for euros on Open Collective. 

> Here's the link: https://opencollective.com/fossify
> 
> This also makes credit/debit card donations possible, thank you!

**Reply by naveensingh** [reply]
I applied, it'll take some time before they approve it.

**Reply by Jozxyqk6** [reply]
Any news on the option in euro?

This is rather essential because the fees for a bank transfer with the banking information provided for dollar amounts are €20, where it would be zero in euro.
